data/campaigns/Son_Of_The_Black_Eye/scenarios/10_Saving_Inarix.cfg

```diff
@@ -189,7 +189,7 @@
 
         [message]
             speaker="Plonk"
-            message= _ "Reinforcements, at last! Whoever you are, you are welcome in Prestim! The fortified bridge over the River Bork was taken this morning by a small commando of elves, we have already tried to assault it twice without success."
+            message= _ "Reinforcements, at last! Whoever you are, you are welcome in Prestim! The fortified bridge over the River Bork was taken this morning by a small commando of elves. We have already tried to assault it twice without success."
```

Comment:
I thought this was fine left as a single sentence - maybe just separate with an em-dash?

Reply:
I would consider the previous form is a [comma splice](https://www.grammarly.com/blog/comma-splice/)

Reply:
What's considered independent? The example they gave regarding koalas being marsupials, I thought it was fine to link them together. Didn't seem independent to me. I admit at this kind of nit-picking level I'm not an 'expert' or anything, even as a native Anglophone. If the experts really think they should be treated independently then okay, let's go with what you've changed.

Reply:
A clause is independent if it has both a subject and a predicate.